lte/gateway/c/core/oai/tasks/amf/amf_Security_Mode.cpp

```diff
@@ -90,10 +90,11 @@ int amf_handle_security_complete_response(
       }
     }
 
-    /* FSM takes care of sending initial context setup request */
-    ue_state_handle_message_initial(COMMON_PROCEDURE_INITIATED1,
-                                    STATE_EVENT_SEC_MODE_COMPLETE, SESSION_NULL,
-                                    ue_mm_context, amf_ctx);
+    // s6a update location request
```

Comment:
Comment can be changed to "Send s6a update location request"

Reply:
addressed